wasmapi/go/datasource.go

```diff
@@ -315,3 +315,6 @@ const (
 	Kind_CString FieldKind = 13
 	Kind_Bytes   FieldKind = 14
 )
+
+// ArgsSeparator is a non-breaking space used to separate function arguments in the "args" field of exec events.
+const ArgsSeparator = string('\u00a0')
```

Comment:
This is specific for the trace_exec gadget, so I think we'll need to define it directly on the gadget.